analysis/interactive.Rmd

```diff
@@ -4,21 +4,265 @@ output:
+Once the file is uploaded, the application will import the `sce` object and initialize the GUI panels with the contents of the object for interactive exploration.
+This type of `iSEE` applications is launched without specifying the `sce` argument, as shown below.
+
+```{r, eval=FALSE}
+iSEE()
+```
+
+## Examples of usages for iSEE apps (#isee-examples)
+
+In the following subsections, we demonstrate some examples of use cases that can be addressed with interactive `iSEE` applications to gain insights into a data set and inform decision-making for downstream analyses.
+In each case, we demonstrate how `iSEE` applications can be preconfigured to launch in a specific state and layout, to facilitate immediate exploration of key aspects of the data set, in a manner most directly relevant to each situation and objectives.
+
+## Examples of usages for iSEE apps
+
+In the following subsections, we demonstrate some examples of use cases that can be addressed with interactive `iSEE` applications to gain insights into a data set and inform decision-making for downstream analyses.
+In each case, we demonstrate how `iSEE` applications can be preconfigured to launch in a specific state and layout, to facilitate immediate exploration of key aspects of the data set, in a manner most directly relevant to each situation and objectives.
+
+We note that these are examples only; we encourage readers to consider them as templates for learning and developing applications adapted to their own specific needs and purposes.
+
+### Quality control
+
+Having previously computed quality control metrics (see section [Quick start](#interactive-quickstart) above), it is possible to launch an app instance that immediately displays a set of panels preconfigured to focus on selected quality control metrics.
+
+For example, a common view for initial inspection of single-cell RNA-sequencing data sets generated by microfluidics platforms plots the library size of each cell, in decreasing order.
+An elbow in this plot generally reveals the transition between good quality cells and low quality cells or empty droplets.
+
+Another common view is to overlay the library size (in log space) as a color gradient over the result of a dimensionality reduction method such as t-SNE, UMAP, or ivis [@vanDerMaaten2008tsne;@mcInnes2018umap;@szubert2019ivis].
+This view may reveal trajectories or clusters associated with library size.
+Depending on the experimental context, such observations may result from technical or biological phenomena.
+For instance, it may indicate that normalization was not completely effective at removing cell biases.
+Alternatively, it could also indicate the presence of multiple cell types that differ in total RNA content.
+
+In the example below, we demonstrate that an `iSEE` app can be preconfigured to immediately display the views described above.
+
+```{r, eval=FALSE}
```

Comment:
I suggest we isolate the set up from the `runApp()`, and only have the latter in an `eval=FALSE` block. This ensures that we verify that the majority of the set-up code is runnable.

Reply:
I've split the relevant chunks in 147c21c